conf/locale/babel_underscore.cfg

```diff
@@ -1,9 +1,5 @@
 # Use this configuration file for extracting strings from .underscore files.
-[underscore: cms/templates/**.underscore]
-input_encoding = utf-8
-[underscore: common/templates/**.underscore]
-input_encoding = utf-8
-[underscore: lms/templates/**.underscore]
+[underscore: **.underscore]
```

Comment:
What file is it? Overall I do trust your judgment.

On Mon, Aug 24, 2015 at 5:15 PM, Andy Armstrong <notifications@github.com>
wrote:

> In conf/locale/babel_underscore.cfg
> <https://github.com/edx/edx-platform/pull/9451#discussion_r37805723>:
>
> > @@ -1,9 +1,5 @@
> >  # Use this configuration file for extracting strings from .underscore files.
> > -[underscore: cms/templates/**.underscore]
> > -input_encoding = utf-8
> > -[underscore: common/templates/**.underscore]
> > -input_encoding = utf-8
> > -[underscore: lms/templates/**.underscore]
> > +[underscore: **.underscore]
>
> I don't understand how that can be the case, but if you are both fine with
> it then I am too.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/edx-platform/pull/9451/files#r37805723>.
>